classes/PayplugLock.php

```diff
@@ -57,14 +57,17 @@ class PayplugLock extends ObjectModel {
 	public function __construct($id = false, $id_lang = false)
 	{
 		parent::__construct($id, $id_lang);
-
-        self::$definition = array(
-       		'table'   => $this->table,
-       		'primary' => $this->identifier,
-       		'fields'  => array(
-       			'id_cart' => array('type' => self::TYPE_STRING, 'validate' => 'isInt'),
-       		),
-       	);
+		
+		if(version_compare(PS_VERSION, 1.5, '>='))
```

Comment:
Are you going to define the table / identifier / fields when the shop run on PrestaShop 1.4 ? These fields exists